third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -217,6 +217,10 @@ instances support `MYSQL_5_5` or `MYSQL_5_6`.
     
 * `root_password` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Initial root password. Required for MS SQL Server, ignored by MySQL and PostgreSQL.
 
+* `source_ip_address` - (Optional, [Beta](https://www.terraform.io/docs/providers/google/guides/provider_versions.html#google-beta)) Public IP address used to connect to and replicate from the [external `MySQL` data source](https://cloud.google.com/sql/docs/mysql/replicatiohttps://cloud.google.com/sql/docs/mysql/replication/replication-from-external#setupn/replication-from-external#setup). Conflicts with `settings` block supports.
```

Comment:
Can you also update docs for any other fields that you changed as part of this PR?

Reply:
Changed description in other fields